src/stories/Library/card-list-page/SearchResultPage.stories.tsx

```diff
@@ -3,7 +3,7 @@ import { SearchResultPage } from "./SearchResultPage";
 import { SearchResultPageSkeleton } from "./SearchResultPageSkeleton";
 
 export default {
-  title: "Blocks / Search Result Page",
+  title: "Blocks / Search Result Page Skeleton",
```

Comment:
We have discussed whether these new titles make sense. I hope to see this addressed in a later commit.